app/src/main/res/layout/fragment_swipeable_pets.xml

```diff
@@ -59,4 +49,14 @@
 
     </LinearLayout>
 
-</RelativeLayout>
+    <com.lorentzos.flingswipe.SwipeFlingAdapterView
+        xmlns:app="http://schemas.android.com/apk/res-auto"
+        android:layout_width="match_parent"
+        android:layout_height="500dp"
+        android:gravity="top|center_horizontal"
+        android:id="@+id/card_view"
+        app:rotation_degrees="16"
+        app:max_visible="4"
+        app:min_adapter_stack="6" />
+
```

Comment:
with frame layouts, the last child of the frame has the highest z-order. So we want our swipe card to be last - after the like / dislike buttons.